views/administration/students.hbs

```diff
@@ -45,37 +45,37 @@
                     </div>
                     <div class="row mt-3 mb-1">
                         <p>
-                            <strong>{{ studentsWithoutConsentCount }} von {{ allStudentsCount }}</strong> Schüler benötigen noch die Einverständniserklärung zur Nutzung der {{theme.short_title}}.
-                            Schüler unter {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} Jahren brauchen zuerst das Einverständnis eines Erziehungsberechtigten, Schüler ab 14 müssen zusätzlich selbst ihr Einverständnis erklären.
+                            <strong>{{ studentsWithoutConsentCount }} von {{ allStudentsCount }}</strong> {{$t "administration.students.text.studentsStillNeedTheDecalaration" }} {{theme.short_title}}.
+                            {{$t "administration.global.text.studentUnder" }} {{CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS}} {{$t "administration.students.text.yearsFirstRequireTheConsent" }}
```

Comment:
maybe as well the paragraph in one single key

Reply:
done ;)